posts/201804.md

```diff
@@ -4,40 +4,53 @@
+The release process has been improved. Instead of announcing a release date and trying to meet it, a Release Candidate 1 (RC1) will now be posted. After it has been tested an bugfixed with the help of the community, a second candidate (RC2) will be released. This is repeated until an RC version with no apparent bugs becomes the final release. The new process removes a ton of pressure from developers and users and gives more time for testing. As our primary consumer-facing product, Decrediton, is growing in features and complexity, more testing will be required for new releases.
+#reasoning:
+#flow & avoids internal conflict by preceding information:
+#A) for Decredition: use bug fix 1.2.1 release
+#B) -> we have a new procedure: 'used to be: ... miss bugs' (obviously we're still missing bugs)
+#
+#if you don't like the suggestivity of 'improved' you can say 'changed'
 
 Politeia is "Getting close to a public beta of voting" ([slack](https://decred.slack.com/archives/C0NRMDVMJ/p1523623105000168)). Decred plugin [merged](https://github.com/decred/politeia/pull/242), paywall and voting are in the testing stage. Ticket voting [works](https://twitter.com/decredproject/status/984518284757221376) on testnet via CLI.
 
+Decred.org received a new sleek [exchanges page](https://www.decred.org/exchanges/).
+
 Trezor support advanced with Decred PR [merged](https://github.com/trezor/trezor-mcu/pull/274). Please note this is only firmware support, to be usable it also needs wallet integration.
 
 WooCommerce [Decred plugin](https://github.com/xifrat/decred-woocommerce-plugin) alpha version is ready for testing.
-
-Website received a new sleek [exchanges](https://decred.org/exchanges/) page and many additions to a list of [contributors](https://decred.org/contributors/).
+ 
+This month, the project has grow by x [contributors](https://www.decred.org/contributors/).
+#comment: would be nice if we could find out a number for that. If its not possible, I would rephrase to: 
+#Plenty of new [contributors](https://www.decred.org/contributors/) have joined the project this month.
+#I strongly prefer this over stating that the contributers list on the website has been updated, because that's too much stating #the obvious imo
 
 Dev activity stats for April: 152 active PRs, 125 commits, 21,656 added and 10,288 deleted lines spread across 7 repositories done by 2-7 developers per repository. ([chart](https://twitter.com/decredproject/status/991734153631563776))
 
 ## ASICs
 
 Updates from Obelisk's Taek:
-
```

Comment:
Blank line before blockquote is standard I think. It somehow works without it, but I would keep it. At least it is present in [GitHub examples](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-text).